docs/workspaces/programming/python-workbench.qmd

```diff
@@ -42,7 +42,7 @@ Simply choose the one that is most convenient to you.
 
 ### Tools are installed in userspace
```

Comment:
I'm wondering if this section is maybe more confusing than actually useful. Do you think it would be better to delete it?

Reply:
Yes, I think you are right, I will try to rephrase it so that people not understanding this can skip reading